test/mockDGXDEX.js

```diff
@@ -0,0 +1,134 @@
+var TestToken = artifacts.require("./mockContracts/TestToken.sol");
+var mockDGXDEX = artifacts.require("./mockContracts/MockDGXDEX.sol")
+var Helper = require("./helper.js")
+
+var myToken;
+
+var ethAddress = '0x00eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee';
+const BigNumber = require('bignumber.js');
+
+contract('MockDGXDEX', function (accounts) {
+    it("create DGX dex and store funds in it.", async function (){
+
+        // create dgx token.
+        dgxToken = await TestToken.new("dex dgx token", "dgx", 9);
+        
```

Comment:
I think the correct way is to declare this in global scope. and all other globals.
using let.
I see its used during test...